api/utils/file_upload_handlers.py

```diff
@@ -0,0 +1,66 @@
+import os, random, string
```

Comment:
could you rename this `api/utils/file_upload_handlers.py`
to  `api/utils/file_handlers.py`
unlike specifically naming it like a function